qiskit/transpiler/target.py

```diff
@@ -238,6 +239,8 @@ def __init__(
         self._gate_name_map = {}
         # A nested mapping of gate name -> qargs -> properties
         self._gate_map = {}
+        # A mapping of number of qubits to set of op names which are global
+        self._global_operations = defaultdict(set)
```

Comment:
How much do we care about reducing search times by keying this on `(num_qubits, num_clbits)`?  I suspect the way you've done this is better in the longer term, since qubits _are_ special compared to clbits, but just throwing out the question in case it was an oversight.

Reply:
In general we haven't been putting any constraints in the target around classical bits. Everything in the target so far has been quantum cosntraints which is why I just used the number of qubits as the dict key here.

Reply:
Sounds good!